src/blog/2023/09/making-sense-of-your-industrial-data.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Making sense of your industrial data
+subtitle: A look at legacy industrial protocol data and making sense of it in an IIoT world.
+description: A look at legacy industrial protocol data and making sense of it in an IIoT world.
+date: 2023-09-08
```

Comment:
Counter proposal.  Because:
1. Modbus is not (really) itself legacy, just that many legacy devices _only_ have one industrial protocol (non (I)IoT) and that protocol is quite often Modbus. (dont want to get in a flame war)
2. The article might get pigeon holed as only being for Modbus and thus 
   1. the graphic should be changed
   2. we lose traffic for those seeking a solution to {name_your_legacy_protocol} types of data
3. The title you propose suggests this is about simply accessing data. Many integrators can do that today with (expensive) OPC packages or HMIs.  The title I was going for was more about "normalisation" of "any" legacy data for an IIoT world.  I.e. not just Modbus but everyone (Serial comms with no protocol, Siemens S7, Omron Fins, Misti MC, Allen Bradley DF1, etc) should be aware there is a solution and we hold the key to it!

```suggestion
title: Modernize your legacy industrial data
subtitle: Working with legacy industrial protocol data from the likes of Modbus and older, non IIoT protocols and putting it to work in an IIoT world.
description: Working with legacy industrial protocol data from the likes of Modbus and older, non IIoT protocols and putting it to work in an IIoT world.
date: 2023-09-11
```